Automattic/wp-calypso
# Changed the command palette icon to the one provided by design

Related to
- https://github.com/Automattic/wp-calypso/issues/89578
- https://github.com/Automattic/wp-calypso/issues/90930
## Proposed Changes

* Changed the icon from a magnifying glass to the one provided by design.
* Changed the tooltip.

## Why are these changes being made?


* The magnifying glass icon doesn't have the expected engagement since users do not associate it with the command palette, by changing the icon we want to increase the engagement.

## Testing Instructions



* Open the Calypso live link
* visit `/sites`
* You should see that the icon corresponds with the new icon instead of the magnifying glass.
* The tooltip should have the updated text corresponding with the new icon.

Before:
![Screenshot 2024-05-22 at 15 53 34](https://github.com/Automattic/wp-calypso/assets/1989914/15151f5c-5b3e-4f0e-bdf4-d6c3fa82a2aa)

After:
![Screenshot 2024-05-22 at 15 53 18](https://github.com/Automattic/wp-calypso/assets/1989914/0feefddb-2816-4805-a48d-b590e634832d)


## Pre-merge Checklist
